Library/Source/NativeEngine.cpp

```diff
@@ -407,7 +411,7 @@ namespace Babylon
         {
             bgfx::reset(w, h, BGFX_RESET_FLAGS);
             bgfx::setViewRect(0, 0, 0, w, h);
-            bgfx::touch(0);
+            bgfx::frame();
```

Comment:
Is this workaround something we should check in? If so, should we create an issue to track it and change it back once we figure out the right way of making this work on Metal?

Reply:
Issue updated here : https://github.com/BabylonJS/BabylonNative/issues/118